models/POP/shell_scripts/cesm2_1/user_datm.streams.txt.CPLHISTForcing.Solar_template

```diff
@@ -0,0 +1,47 @@
+<dataSource>
+    Raeder et al. 80 member CAM/DART reanalysis
+</dataSource>
```

Comment:
Ben ... please look at 
/glade/work/thoar/git/DART_CLM/models/clm/shell_scripts/cesm2_0/datm.streams.txt.CPLHISTForcing.Solar_complete
for an example of what I'd like to see in these.   When we were able to curate the files in the RDA, it became a lot more sensible to reference them there - and I added some curation information. I know the POP stream templates are very similar to what was in the repository, but now seems like a good time to update them.

Reply:
No problem.